### DISCUSSION TITLE
assets loading issue with (www.)

### DISCUSSION THREAD

**Discussion by Guellaf** [thedevdojo/voyager#5497] [Q&A]
### Laravel version

8.65

### PHP version

7.4.9

### Voyager version

1.5

### Database

Version information: 5.0.2

### Description

https://_**www**_.example.com/admin 
loader infinity spinning 
but https://example.com/admin  works perfect 

### Steps to reproduce

??

### Expected behavior

??

### Screenshots

![image](https://user-images.githubusercontent.com/47402966/143868896-16aebc73-68b9-4687-991a-1a9909bff77f.png)

**Comment by fletch3555** [comment]
We can't help without more information, so answering "??" to those questions is not adequate.  There's a reason they're required.

Having said that, if the only difference is that it works with the bare domain but not with www, then that's very much an issue with your webserver or app config, not a bug with Voyager.

**Reply by Guellaf** [reply]
im sorry ! my english is not native so i dont get it 
can you explain those ( Steps to reproduce , Expected behavior ) translating programming words  to arabic is not god idea

**Reply by Guellaf** [reply]
other framework and php native code works perfect in the same domain and same host same webserver with or without www.

**Reply by Guellaf** [reply]
I test same code in domains without dash (-) and work perfectly because my domain name has dash 
so any idea ?
